codedoor/templates/codedoor/edit_application.html

```diff
@@ -5,37 +5,38 @@
 
 <head>
   <link rel="stylesheet" href="{% static 'css/application.css' %}">
+  <link rel="stylesheet" href="{% static 'css/modal.css' %}">
 </head>
 <form action="{% url 'codedoor:edit_application' pk=pk %}" method="POST">
     {% csrf_token %}
     <div class="entry">
         <h4>Position:</h4><br>
-        <input name="position" value="{{position}}" class="text"></input><br><br>
+        <input name="position" value="{{position}}" type="text"></input><br><br>

```

Comment:
Can we remove the `<br>` tags and do spacing through margins only? Also prefer `<input />` to `<input></input>`.